openapi.yaml

```
@@ -15528,8 +15532,11 @@ components:
           example: E1AF5EEC-526F-487D-B317EBEB34C87D71
         zip:
           type: string
-          description: The zip code of this Account's billing address.
-          maxLength: 16
+          description: |
+            The zip code of this Account's billing address. The following restrictions apply:
+
+            - May only consist of letters, numbers, spaces, and hyphens.
+            - Must not contain more than 9 letter or number characters.
           example: 19102
```

```suggestion
          example: 19102-1234
```
Can this be changed to a more complicated example, to show a special character usage?